src/clients/SpokePoolClient.ts

```diff
@@ -835,6 +836,15 @@ export class SpokePoolClient extends BaseAbstractClient {
           ? { ...(spreadEventWithBlockNumber(event) as V3FillWithBlock), destinationChainId: this.chainId }
           : { ...(spreadEventWithBlockNumber(event) as V2FillWithBlock) };
```

Comment:
Why doesn't spreadEvent work? Shouldn't we put this logic in there if its broken?

Reply:
I'm working on a larger fix to see why `spreadEvent` isn't working. This is a patch - the next PR will be a fix